server/utils/referralUtils.ts

```diff
@@ -51,6 +51,26 @@ export default class ReferralUtils {
     return referral.hasReviewedProgrammeHistory && referral.oasysConfirmed && !!referral.additionalInformation
   }
 
+  static programmeSummaryListRows(
+    coursePresenter: CoursePresenter,
+    organisationName: Organisation['name'],
+  ): Array<GovukFrontendSummaryListRowWithValue> {
+    return [
+      {
+        key: { text: 'Programme name' },
+        value: { text: coursePresenter.nameAndAlternateName },
+      },
+      {
+        key: { text: 'Programme strand' },
+        value: { text: coursePresenter.audiences.map(audience => audience.value).join(', ') },
```

Comment:
Will keep it separate for now as I have a question on my to do list to ask design if in fact they should be the same content anyway, since it's very similar.